news/13048.process.rst

```diff
@@ -0,0 +1,3 @@
+Started releasing to PyPI from a GitHub Actions CI/CD workflow that implements trusted publishing and bundles :pep:`740` digital attestations.
+
+In addition to being signed, the released distribution packages are now reproducible through the commit timestamp.
```

Comment:
Sorry, I should've flagged this directly. This is currently inaccurate due to the setuptools bug I mentioned. I think it may be better to simply drop this until we confirm with setuptools that they're reproducible-ready.

Reply:
@sbidoul, are we leaving this in the release notes? The source distributions are not reproducible yet, and I'd rather not claim pip is fully reproducible until we actually are. The reproducible crowd, although probably small, can be pretty loud.

P.S. I plan on looking through and copyediting the news entries tonight so we're ready to cut a release.

Reply:
@ichard26 Sorry I had missed your comment. We can update it to mention wheels only.

Reply:
Or drop it indeed, because I have not compared a wheel built on different platforms.